ironfish/src/network/peers/peerManager.ts

```diff
@@ -782,55 +793,54 @@ export class PeerManager {
     }
   }
 
-  start(): void {
-    this.distributePeerListHandle = setInterval(() => this.distributePeerList(), 5000)
-    this.disposePeersHandle = setInterval(() => this.disposePeers(), 2000)
+  async start(): Promise<void> {
+    await Promise.allSettled([
```

Comment:
This `Promise.allSettled` isn't needed, none of these functions involve promises.

Reply:
It looks like this was copied from the other parts of our code base without reading what it does, because what promises is it waiting to settle on? If you aren't sure what a piece of our code is doing, you can always check the docs and play around with it locally before copying it. Check out the docs on `Promise.allSettled` here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled